### DISCUSSION TITLE
Povratna informacija na prijavu projekta

### DISCUSSION THREAD

**Discussion by MarkoMijac** [foivz/rpp23-project-lpamer20-mfranjic21-msasko1#1] [General]
Poštovani kolege,

sama domena je u redu, međutim potrebno je doraditi funkcionalnosti, odnosno definirati realne funkcionalnosti za vašu domenu iznajmljivanja vozila. Npr. Izbornik nije funkcionalnost. Predlažemo da analizirate neku od postojećih aplikacija za rent-a-car i da vam to bude inspiracija za smišljanje realnih funkcionalnosti vaše aplikacije. Pri tome, s obzirom da radite istu domenu i na kolegiju RAMPU, u okviru kolegija RPP ćete izraditi desktop aplikaciju koja će za korisnike imati vlasnika/zaposlenike rent-a-car poduzeća. Sukladno tome, vaše funkcionalnosti u ovom repozitoriju trebaju biti fokusirane na administriranje flote vozila, provedbu procesa iznajmljivanja vozila, i sl. iz perspektive gore navedenih korisnika. Nasuprot tome na kolegiju RAMPU ćete izraditi mobilnu aplikaciju koja će se fokusirati na funkcionalnosti krajnjeg korisnika/kupca (više detalja ćete dobiti od nastavnika na tom kolegiju).

Kada doradite vašu prijavu sa novim funkcionalnostima molim da me obavijestite odgovorom na ovu poruku.

Lijep pozdrav!

**Comment by lpamer20** [comment]
Poštovani profesore,

Promijenili smo neke od funkcionalnosti tako da smo neke zaokružili a neke prilagodili samo da se tiču administratora pa nas obavijestite ako još nešto bude trebalo.

Lijep pozdrav.

**Reply by MarkoMijac** [reply]
Poštovani kolege,

vašu prijavu bi trebalo još doraditi. Naime, neke od funkcionalnosti mi se čine "na silu" ugurane u prijavu, npr. Postavke, Podrška, Ispis u pdf, i sl. S druge strane fale neke sržne funkcionalnosti kao što su izdavanja vozila korisniku, preuzimanja vozila od korisnika, izdavanje računa, i sl. Funkcionalnost ispis u pdf je dio funkcionalnosti izdavanja računa.

Lijep pozdrav!

**Reply by lpamer20** [reply]
Poštovani,

Ispis u PDF je implenetiran u funkcionalnost "Pregled Financijskih transakcija" kao sve vezano za račune te je dodana nova funkcionalnost "Status vozila" koja će ispisivat stanje o pojedinom vozilu, kad, gdje se treba zaprimiti, kad se vraća itd, S tim da bi se status vozila orijentirao i na implementaciju mape na kojoj će biti vidljive lokacije o preuzimanju i vraćanju.

Lijep pozdrav!

**Reply by MarkoMijac** [reply]
Poštovani kolege,

* F06 - Može, ali prijevodi moraju biti ili u bazi ili u zasebnim datotekama. Kada se korisnik logira onda se u ovisnosti na to koji je jezik odabrao dinamički učitavaju prijevodi.
* F09 i F10 nisu baš najbolje i najrealnije funkcionalnosti. Ako ćete ih zadržati pokušajte ih malo proširiti.

Uzmite u obzir gore navedene komentare i započnite s radom.

Lijep pozdrav!
